Controller/Adminhtml/Exportcategory/Export.php

```diff
@@ -49,50 +50,61 @@ public function __construct(
         \Magento\Framework\Controller\Result\RawFactory $resultRawFactory,
         \Magento\Framework\App\Response\Http\FileFactory $fileFactory,
         \Magento\Backend\App\Action\Context $context
-    ) {
-    
+    )
+    {
+
         $this->_resultForwardFactory = $resultForwardFactory;
         $this->_storeManager = $storeManagerInterface;
         $this->_categoryFactory = $categoryFactory;
         $this->_productcollection = $prodcollection;
         $this->resultRawFactory = $resultRawFactory;
-        $this->fileFactory  = $fileFactory;
+        $this->fileFactory = $fileFactory;
         parent::__construct($context);
     }
 
     public function execute()
     {
         $store_id = $this->getRequest()->getPost('store_id');
-        $singlestoremode = $this->_storeManager->isSingleStoreMode();
+        $singleStoreMode = $this->_storeManager->isSingleStoreMode();
         $_stores = [];
-        if (!$singlestoremode) {
+        if (!$singleStoreMode) {
             $stores = $this->_storeManager->getStores();
             foreach ($stores as $key => $store) {
                 $_stores[$store->getId()] = $store->getCode();
             }
         }
-         $fileName = 'categories.csv';
-        $content = '"category_id","parent_id"';
-        $content .= ',"store"';
-        $content .= ',"name","path","image"';
-        $content .= ',"url_key"';
-        $content .= ',"is_active","is_anchor","include_in_menu","meta_title","meta_keywords","meta_description","display_mode","custom_use_parent_settings","custom_apply_to_products","custom_design","custom_design_from","custom_design_to","default_sort_by","page_layout","description","products"'."\n";
+
+        $fileName = 'categories.csv';
+
+        $content = '"category_id","parent_id","store","name","path","image","url_key","is_active","is_anchor"';
```

Comment:
this might be better stored as an array for easier viewing, then using implode to get the string value